packages/pilot/src/formatters/paymentLinkSpec.js

```diff
@@ -23,6 +30,30 @@ const expiresIn = ({
   return parseInt(duration.asMinutes(), 10)
 }
 
+const checkFeePayerAndSetFreeInstallments = (object) => {
+  if (object.fee_payer === 'customer') {
+    return minInstallments
+  }
+
+  return parseIntValue('free_installments', object)
+}
+
+const checkChargeTransactionAndSetMaxInstallments = (object) => {
+  if (object.chargeTransactionFee) {
+    return maxInstallments
+  }
+
+  return parseIntValue('max_installments', object)
+}
+
+const checkChargeTransactionAndSetInterestRate = (object) => {
+  if (object.chargeTransactionFee) {
+    return 0
+  }
+
+  return parseIntValue('interest_rate', object) || 0.01
```

Comment:
acho uma boa já retornarmos zero aqui, pois da forma que está, estamos setando pelo lojista uma taxa de juros sem avisa-lo, caso eu avance os steps sem preencher o interest_rate

```suggestion
  return parseIntValue('interest_rate', object) || 0
```